stubs/dateparser/dateparser/date.pyi

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Optional
+
+from dateparser.conf import apply_settings as apply_settings, check_settings as check_settings
```

Comment:
The syntax is a bit weird, but `from foo import bar as bar` means that `bar` should be importable not only from `foo`, but also from the current module.

This line says that you can do `from dateparser.date import apply_settings`. It seems to be something that stubgen generated but we don't really want. Apparently `apply_settings` is not documented at all, so the stubs shouldn't say that it is also available in a different module.

Can you delete this and other similar imports?

Reply:
> should be importable not only from foo, but also from the current module.

May I know please, is there any particular ways to quickly detect when?

Reply:
If there is documentation for the library, it usually shows how the functions are meant to be imported.

Reply:
> Can you delete this and other similar imports?

First, I just removed the as part. Then I was obliged to remove all the imports as those imports were not used. I'll have to learn a little bit more about this construct (in the context of type hinting).  

I think it should be good (at least for now :p)